app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateCreditMemoTotalRefundFullInvoicePartialShipmentGroupedProductCustomStock.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminCreateCreditMemoTotalRefundFullInvoicePartialShipmentGroupedProductCustomStock" extends="AdminCreateCreditMemoTotalRefundFullInvoiceFullShipmentGroupedProductDefaultStock">
+        <annotations>
+            <stories value="Grouped Product Custom Stock."/>
+            <title value="Create Credit Memo with Total Refund after Full Invoice and Partial Shipment with Grouped product on Custom stock."/>
+            <description value="Verify, admin able to create credit memo with total refund after full invoice and partial shipment with grouped product on custom stock."/>
+            <testCaseId value="https://app.hiptest.com/projects/69435/test-plan/folders/735182/scenarios/2553867"/>
```

Comment:
`AdminCreateCreditMemoTotalRefundPartialShipmentGroupedProductCustomStock`

Reply:
Not able to find AdminCreateCreditMemoTotalRefundPartialShipmentGroupedProductCustomStock on 2.3-develop